client/pom.xml

```diff
@@ -0,0 +1,153 @@
+                <groupId>org.apache.netbeans.utilities</groupId>
+                <artifactId>nbm-maven-plugin</artifactId>
+                <version>4.8</version>
+                <extensions>true</extensions>
+                <configuration>  
+                    <licenseFile>LICENSE</licenseFile>
+                    <licenseName>Apache 2.0</licenseName>
+                    <moduleDependencies>
+                        <dependency>
+                            <id>org.netbeans.api:org-netbeans-modules-lsp-client</id>
+                            <type>impl</type>
+                        </dependency>
+                    </moduleDependencies>                   
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.11.0</version>
+                <configuration>
+                    <source>17</source>
+                    <target>17</target>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <version>3.3.0</version>
+                <configuration>
+                    <archive>
+                        <manifestFile>${project.build.outputDirectory}/META-INF/MANIFEST.MF</manifestFile>
+                    </archive>
+                </configuration>
+            </plugin>
+            <!--Download Camel LSP server-->
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>get-deps</id>
+                        <goals>
+                            <goal>copy</goal>
+                        </goals>
+                        <phase>generate-resources</phase>
+                    </execution>
+                </executions>
+                <configuration>
+                    <artifactItems>
+                        <artifactItem>
+                            <groupId>com.github.camel-tooling</groupId>
+                            <artifactId>camel-lsp-server</artifactId>
+                            <version>${camel-lsp-server-version}</version>
+                        </artifactItem>
+                    </artifactItems>
+                    <useBaseVersion>true</useBaseVersion>
+                    <skip>false</skip>
+                    <outputDirectory>${basedir}/src/main/resources/</outputDirectory>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+    <repositories>
+        <repository>
+            <id>apache.snapshots</id>
+            <name>Repository hosting NetBeans modules</name>
+            <url>https://repository.apache.org/content/repositories/snapshots/</url>
```

Comment:
Is the Apache Snapshots really hosting netbeans modules?

Also wondering if this snapshot repository is really used